### ISSUE TITLE
Team update - Jul 19, 2023

### ISSUE THREAD

**Issue by trallard** [Quansight-Labs/jupyter-a11y-mgmt#228]
This is a @Quansight-Labs/czi-a11y-grant team sync 🎉🎉🎉! This is a way for the Team Members to provide status reports on what they've been up to this week and request help and attention for things they are working on. This issue will be closed at the end of the day.

_Copy and paste the template below, and answer questions as you wish!_

<details>
<summary>Response Template</summary>

```md
**Thanks I'd like to give 🙌**
- So-and-so helped me out a lot with foo...

**Updates from last week :heavy_check_mark:**
- I worked towards deliverable: <link-to-deliverable>

**Challenges I faced and things I'd like assistance with 🙏**
- I had a hard time figuring out ...

**My availability for next week**
- I'll be off on foo day...

**Important items for discussion 💬**
- I have a question about goal <link-to-goal>
- Can @foo give a comment on issue #NN ?
- I opened #NN for discussion, please chime in
- I need to discuss foo
```

</details>

#### :mag: Needs Triage

_The [Needs Triage issues](https://github.com/Quansight-Labs/jupyter-a11y-mgmt/labels/status%3A%20needs%20triage) require an initial assessment and labeling._

No issues need triage! 🎉

### :dart: Project boards

Please make sure the boards reflect the current status of the project.

[Jupyter a11y grant project](https://github.com/orgs/Quansight-Labs/projects/5/views/5)

**Comment by isabela-pf** [comment]
**Thanks I'd like to give 🙌** 

**Updates from last week :heavy_check_mark:**

- #184 [I replied to more review](https://github.com/Quansight-Labs/JupyterLab-user-testing/pull/10). There are still a few comments I need to address I think, but I'm not sure how I want to do it at the time of writing.
- #178 JupyterLab 3.4.4 [accessibility statement has a PR to the JupyterLab docs](https://github.com/jupyterlab/jupyterlab/pull/14856). Thank you to @steff456 for giving this a look since I haven't had to add to JupyterLab docs so completely before.
- I presented on notebook authoring best practices. Happy to answer any questions about that if you have them.

**Challenges I faced and things I'd like assistance with 🙏**
- I don't have anything right now.

**My availability for next week**
- I'll be here all week.

**Important items for discussion 💬**
- Are we planning to have a post-grant discussion to talk about what we think worked and what we might want to do differently in the future?

**Comment by trallard** [comment]
> Are we planning to have a post-grant discussion to talk about what we think worked and what we might want to do differently in the future?

Yes we should, will only happen in August as we have now 2 weeks left to wrap up stuff 

Things that would be good to discuss while I am missing this:

1. How are we doing for deliverables? 
2. What help do you need from me?

From top of my head - @steff456  only straggling deliverable is getting the GH dark PR merged which is top of my TODO for tomorrow

**Comment by gabalafou** [comment]
**Thanks I'd like to give 🙌**
- @steff456 for agreeing to cross-check PR https://github.com/Quansight/peri-store-frontend/pull/62 with https://github.com/Quansight-Labs/jupyterlab-accessible-themes/issues/59

**Updates from last week :heavy_check_mark:**
- Responding to review on https://github.com/jupyterlab/jupyterlab/pull/14115

**Challenges I faced and things I'd like assistance with 🙏**
- I had a hard time figuring out why sidebars can be tabbed when installing Jupyter Lab (v4.0.3) via Conda but not when building source from latest `main` branch.

**My availability for next week**
- I'll be off on Friday, July 28 through Monday, July 31 (four-day weekend)

**Comment by steff456** [comment]
**Updates from last week :heavy_check_mark:**
- Wrapped up the theme extension work:
    - Made changes to GH dark https://github.com/Quansight-Labs/jupyterlab-accessible-themes/pull/53 
    - Fix some issues in the `pyproject.toml` and readme - https://github.com/Quansight-Labs/jupyterlab-accessible-themes/pull/67
    - Made changes to Pitaya Smoothie and GH light themes - https://github.com/Quansight-Labs/jupyterlab-accessible-themes/pull/62
- Check build of Isabela's documentation PR

**Challenges I faced and things I'd like assistance with 🙏**
- I'm ok

**My availability for next week**
- I'll be here

**Important items for discussion 💬**
- what will be our time allocation for the following week?

**Comment by isabela-pf** [comment]
To answer

> What help do you need from me?

- @steff456 needs review on [GitHub dark PR](https://github.com/Quansight-Labs/jupyterlab-accessible-themes/pull/53)
- @steff456 needs review (and merge?) on [audit PR](https://github.com/jupyter/accessibility/pull/133)
- @gabalafou help figuring out about the aforementioned "I had a hard time figuring out why sidebars can be tabbed when installing Jupyter Lab (v4.0.3) via Conda but not when building source from latest main branch." (optional).
- @isabela-pf doesn't need anything at the moment because she needs to reply to some existing review.

For 

> How are we doing for deliverables?

We feel like most of the answers are in the updates. Tasks are still moving rather than blocked. I feel like all the things I'm seeing in the to do section of the project board has been there for a bit, and they are not ones we've recently discussed. This makes me think everything we have discussed for the end of grant is at least in progress. 

Is that a helpful answer?
